sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Release History
+
+## 1.0.0 (Unreleased)
```

Comment:
Some background:
`@azure/test-utils-recorder` is not a published package and doesn't follow the versioning, it is only used as a utility for tests in the other SDKs in this repository.
Though we update the changelog with changes, the version would forever remain 1.0.0 and the package would be unreleased.

That being said, I see no harm in adding this header.